src/fonduer/parser/visual_linker.py

```diff
@@ -16,6 +16,9 @@
 from fonduer.parser.models import Sentence
 from fonduer.utils.utils_visual import Bbox
 
+# Define a type alias for readability
+WordId = Tuple[int, int]
```

Comment:
The more type aliases we use, the more I think we should have a place to store all these.

Reply:
For the record, this comment is related to https://github.com/HazyResearch/fonduer/pull/429#issuecomment-635218428.

Reply:
In general, I agree with having `fonduer.typing` module where you define type aliases.
Let's create an issue to address this.